Arrowgene.Ddon.Database/Files/Database/Script/AddMissingOnDeleteCascades_migration.sql

```diff
@@ -0,0 +1,88 @@
+﻿PRAGMA foreign_keys=OFF;
```

Comment:
Probably SQLite specific 😭

Reply:
Added some rough equivalents for PSQL & MariaDB. Untested though.

```
schema = Regex.Replace(schema, @"PRAGMA(\s*)foreign_keys=(\s*)(OFF|0)(\s*);", "SET session_replication_role='replica';", RegexOptions.IgnoreCase);
schema = Regex.Replace(schema, @"PRAGMA(\s*)foreign_keys=(\s*)(ON|1)(\s*);", "SET session_replication_role='origin';", RegexOptions.IgnoreCase);
```
            
```
schema = Regex.Replace(schema, @"PRAGMA(\s*)foreign_keys=(\s*)(OFF|0)(\s*);", "SET FOREIGN_KEY_CHECKS=0;", RegexOptions.IgnoreCase);
schema = Regex.Replace(schema, @"PRAGMA(\s*)foreign_keys=(\s*)(ON|1)(\s*);", "SET FOREIGN_KEY_CHECKS=1;", RegexOptions.IgnoreCase);
```